feed/tests/test_documentationview.py

```diff
@@ -206,3 +218,83 @@ def test_delete_documentation_normal_user(self):
+        wflvl2_1 = factories.WorkflowLevel2(workflowlevel1=wflvl1_1)
+        WorkflowTeam.objects.create(workflow_user=self.tola_user,
+                                    workflowlevel1=wflvl1_1)
+        documentation1 = factories.Documentation(name='Document 1',
+                                                 workflowlevel1=wflvl1_1,
+                                                 workflowlevel2=wflvl2_1)
+
+        another_org = factories.Organization(name='Another Org')
+        wflvl1_2 = factories.WorkflowLevel1(
+            name='WorkflowLevel1_2',
+            organization=another_org)
+        wflvl2_2 = factories.WorkflowLevel2(workflowlevel1=wflvl1_2)
+        WorkflowTeam.objects.create(workflow_user=self.tola_user,
+                                    workflowlevel1=wflvl1_2)
+        factories.Documentation(name='Document 2', workflowlevel1=wflvl1_2,
+                                workflowlevel2=wflvl2_2)
+
+        request = self.factory.get(
+            '/api/documentation'
+            '/?workflowlevel2__workflowlevel1__organization__id=%s' %
+            self.tola_user.organization.pk)
+        request.user = self.tola_user.user
+        view = DocumentationViewSet.as_view({'get': 'list'})
+        response = view(request)
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(len(response.data), 1)
+        self.assertEqual(response.data[0]['name'], documentation1.name)
+
+    def test_filter_documentation_org_org_admin(self):
+        group_org_admin = factories.Group(name=ROLE_ORGANIZATION_ADMIN)
+        self.tola_user.user.groups.add(group_org_admin)
+
+        country1 = factories.Country()
+        wflvl1_1 = factories.WorkflowLevel1(
+            name='WorkflowLevel1_1',
+            country=[country1],
+            organization=self.tola_user.organization)
+        wflvl2_1 = factories.WorkflowLevel2(workflowlevel1=wflvl1_1)
+        WorkflowTeam.objects.create(workflow_user=self.tola_user,
+                                    workflowlevel1=wflvl1_1)
+        documentation1 = factories.Documentation(name='Document 1',
+                                                 workflowlevel1=wflvl1_1,
+                                                 workflowlevel2=wflvl2_1)
+
+        country2 = factories.Country(country='Brazil', code='BR')
+        wflvl1_2 = factories.WorkflowLevel1(
+            name='WorkflowLevel1_2',
+            country=[country2],
+            organization=self.tola_user.organization)
+        wflvl2_2 = factories.WorkflowLevel2(workflowlevel1=wflvl1_2)
+        WorkflowTeam.objects.create(workflow_user=self.tola_user,
+                                    workflowlevel1=wflvl1_2)
+        factories.Documentation(name='Document 2', workflowlevel1=wflvl1_2,
+                                workflowlevel2=wflvl2_2)
+
+        request = self.factory.get(
+            '/api/documentation'
+            '/?workflowlevel2__workflowlevel1__country__country=%s' %
```

Comment:
It's a pity that someone needs to know so much about the internal details and relations of a model to filter properly in the API. But yeah, that's life...

Reply:
I think its possible to change but we need to create FilterSet for them. We check it later.